flake.nix

```diff
@@ -60,8 +60,18 @@
       flake = false;
     };
 
-    fourmolu-012 = {
-      url = "https://hackage.haskell.org/package/fourmolu-0.12.0.0/fourmolu-0.12.0.0.tar.gz";
+    # smunix: github:haskell/hie-bios defines
```

Comment:
Here's a more detailed explanation of the reasons why I had to patch `implicit-hie-cradle`

Reply:
PRs welcome on implicit-hie* repos. Saddly I don't have much time for Haskell these days. If your 0.5 branch is ready make a PR and I'll publish that to hackage tomorrow.

Reply:
Will give it another week; we may need to coordinate with a separate PR for a new version of HLS. That effort is still ongoing